### PR TITLE
[Simplified login] Not WordPress.com account screen changes

### LINKED ISSUE #7612: [Simplified login] Update the "WordPress account not found" screen
Override the screen in the app to allow adding a “Create an account” button.

### PR DESCRIPTION
**Pull request by hichamboushaba**
Closes: #7612

Please don't merge until #7600 is merged.

### Description
This PR handles the changes of the "Not WP account" screen for simplified login.

### Images/gif
https://user-images.githubusercontent.com/1657201/198046898-4180b024-254d-4637-aab0-c8ca1e00c6b1.mov

- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -2545,7 +2548,7 @@
     <string name="enter_your_site_address">Enter your site address</string>
     <string name="reset_your_password">Reset your password</string>
     <string name="signup_confirmation_title">Signup confirmation</string>
-    <string name="what_is_wordpress_link">&lt;u&gt;What is WordPress.com?&lt;/u&gt;</string>
+    <string name="what_is_wordpress_link">What is WordPress.com?</string>
```
Comment (hichamboushaba) [inline WooCommerce/src/main/res/values/strings.xml:2574]: I removed the underline from this key and `login_need_help_finding_email`, we don't have the underline in Figma, and I confirmed that the last decision was to remove even before this project (ref p1658816607458229/1658117084.869099-slack-C0354HSNUJH), I think it was just missed.

**Comment by hafizrahman** [comment]
I got a crash when testing this from using the "Continue with WordPress.com" button on the Prologue screen. `remoteConfig.getString(SIMPLIFIED_LOGIN_VARIANT_KEY).uppercase()` in `FirebaseRemoteConfigRepository#getSimplifiedLoginVariant()` was returning an empty string. 

~So I merged latest trunk to this branch to include the fix from #7642~

Update: Thanks @hichamboushaba for resetting that, didn't realize this isn't targeting trunk 🤦🏼

**Review by hafizrahman — CHANGES_REQUESTED** [review]
Minor:

1. Can you check the "no WP.com account" screen on your side? Mine shows a gray instead of white background, for some reason 🤔

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/sitepicker/SitePickerViewModel.kt [hunk 2]
```diff
@@ -116,6 +116,11 @@ class SitePickerViewModel @Inject constructor(
             }
             fetchSitesFromApi(sitesInDb.isEmpty() || !loginSiteAddress.isNullOrEmpty())
         }
+        if (sitePickerViewState.userInfo == null) {
+            launch {
+                fetchUserAccount()
```
Comment (hafizrahman) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/sitepicker/SitePickerViewModel.kt]: ❓  I haven't followed the entire Site Picker logic so probably a basic question: Why is it needed to fetch user account here, while `updateSiteViewDetails()` is called before it and that function already does `getUserInfo()`? 

Would it be better if `updateSiteViewDetails()` does the fetch instead if it gets no result for `getUserInfo()`?

**Comment by hafizrahman** [comment]
In general everything works as expected to me, just left two small questions above.

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/sitepicker/SitePickerViewModel.kt (hunk 2 above)
Comment (hichamboushaba) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/sitepicker/SitePickerViewModel.kt]: I didn't have any specific reason for putting it there, I just wanted to put it next to where we fetch sites.
I made a small change in e4e1a928008e4d50c9968b2c542e910f0d6213d2, I hope it makes things clearer.

**Comment by hichamboushaba** [comment]
>  Can you check the "no WP.com account" screen on your side? Mine shows a gray instead of white background, for some reason 🤔

@hafizrahman I believe this happens only on the emulator, maybe because of using the mouse which triggers some sort of selection.
Can you double-check if happens in real devices?

**Comment by hafizrahman** [comment]
> I believe this happens only on the emulator, maybe because of using the mouse which triggers some sort of selection.

Huh, you're right. I've never run into this before for some reason 🤔

**Review by hafizrahman — APPROVED** [review]

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/sitepicker/SitePickerViewModel.kt (hunk 2 above)
Comment (hafizrahman) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/sitepicker/SitePickerViewModel.kt]: That's awesome, thanks for adding that change!

**Pull request merged**